content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -94,6 +95,20 @@ variables (map of keys)
     └── sensitive (boolean)
 ```
 
+### Value: `target_addrs`
+
+* **Value Type:** An array of strings representing [resource addresses](/docs/internals/resource-addressing.html).
+
+Provides the targets specified using the [`-target`](/docs/commands/plan.html#resource-targeting) flag in the CLI or the `target-addrs` attribute in the API.
+
+To prohibit targeted runs altogether, check that the length of this collection is zero:
```

Comment:
I think this might be null or []. Will this go boom if its null?

(will try it out locally and return with an answer!)

Reply:
Yeah. I think this needs to account for null being a valid option. Here's the result for a non-targeted run with this rule:

![Screen Shot 2020-05-20 at 10 15 54 AM](https://user-images.githubusercontent.com/2732204/82463894-06945880-9a83-11ea-95a2-387c03a360cb.png)

Reply:
:sad_trombone: thanks for testing that!

Reply:
The updated example should better account for `undefined`.